pkg/chartutil/values.go

```diff
@@ -288,6 +289,35 @@ func coalesceTables(dst, src map[string]interface{}) map[string]interface{} {
 	return dst
 }
 
+// ReleaseOptions represents the additional release options needed
+// for the composition of the final values struct
+type ReleaseOptions struct {
+	Name      string
+	Time      *timestamp.Timestamp
+	Namespace string
+}
+
+// ToRenderValues composes the struct from the data coming from the Releases, Charts and Values files
+func ToRenderValues(chrt *chart.Chart, chrtVals *chart.Config, options ReleaseOptions) (Values, error) {
```

Comment:
LGTM. One suggestion is to use 'time.Time' in lieu of timestamp.Timestamp in ReleaseOptions. Use github.com/golang/protobuf/ptypes.TimestampProto function to convert to google.protobuf.Timestamp.